inst/extdata/load_evaporation.R

```diff
@@ -10,60 +10,60 @@ if (FALSE)
 {
   # Base URL to potential evaporation files on DWD server
   base_url <- kwb.dwd:::ftp_path_cdc("grids_germany/monthly/evapo_p")
-  
+
   # List data files
   relative_urls <- grep(
     "\\.asc\\.gz$", kwb.dwd::list_url(base_url), value = TRUE
   )
-  
+
   # Provide full paths
   urls <- file.path(base_url, relative_urls)
 
-  # Read all files into a list of matrices  
+  # Read all files into a list of matrices
   evaporation_matrices <- lapply(urls, read_evaporation_matrix_from_url)
 
-  # Provide metadata: file name, year, month  
+  # Provide metadata: file name, year, month
   file_info <- data.frame(
     file = relative_urls,
     year = sapply(evaporation_matrices, kwb.utils::getAttribute, "year"),
     month = sapply(evaporation_matrices, kwb.utils::getAttribute, "month")
   )
-  
+
   head(file_info)
-  
+
   str(evaporation_matrices[[1]])
-  
+
   # Get Berlin matrix, same size as DWD evpo matrix (Berlin grid cells set to 1, rest of cells = NA)
-    Berlin_DWD_mask <- Berlin_DWD_mask()
-  
+    berlin_dwd_mask <- berlin_dwd_mask()
```

Comment:
I suggest to give the function a name that is different from the name of the variable that you assign the function's value to. Also it is good practice to give a function a name that represents an English verb, not a noun.
```suggestion
    berlin_dwd_mask <- get_berlin_dwd_mask()
```